src/EditorFeatures/CSharpTest/CodeActions/EncapsulateField/EncapsulateFieldTests.cs

```diff
@@ -776,12 +789,17 @@ class Program
 }
 ";
 
-            await TestActionCountAsync(text, 2);
+            await TestActionCountAsync(text, 2, GetRemoteHostOptions(host));
+        }
+
+        private TestParameters GetRemoteHostOptions(TestHost host)
+        {
+            return new TestParameters(options: Option(RemoteHostOptions.RemoteHostTest, host != TestHost.InProcess));
```

Comment:
good question.  The basic reason (copied from elsewhere) is that this means if we add more out-of-proc options (which happens in some case) then all of them will run OOP since they're *not* the .InProcess value.